src/Language/Haskell/TH/DeepStrict.hs

```diff
@@ -73,6 +74,14 @@ emptyContext = do
       , contextRecursionDepth = 1000
       }
 
+defaultContextOverride :: M.Map TH.Name (Maybe [Strictness])
+defaultContextOverride = M.fromList
```

Comment:
change name to reflect that these aren't overriding anything, they can just be `baseOverride`